nav2_behavior_tree/include/nav2_behavior_tree/bt_cancel_action_node.hpp

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) 2022 Pradheep Padmanabhan, Neobotix GmbH
```

Comment:
Requires test coverage.

Also, I think (?) the copyright is probably to your employer if you did this on company time / resources. We assign by copyright holder not author (so mine all say Samsung Research)